src/test/java/se/sundsvall/supportmanagement/integration/db/ErrandsRepositoryTest.java

```diff
@@ -127,6 +128,47 @@ void findByFilter(String filter) {
 				tuple("ERRAND_ID-1", "ASSIGNED_GROUP_ID-1", "ASSIGNED_USER_ID-1"));
 	}
 
+	@Test
+	void findByDateFilter() {
+		//Setup date filter
+		final var dateFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSXXX");
+		final var from = OffsetDateTime.now().minusMinutes(1).format(dateFormat);
+		final var to = OffsetDateTime.now().plusMinutes(1).format(dateFormat);
+		final var filter = String.format("(created > '" + from + "' AND created < '" + to + "')");
+
+		Specification<ErrandEntity> specification = new FilterSpecification<>(filter);
+		Pageable pageable = PageRequest.of(0, 20);
+
+		//Setup ErrandEntity to find
+		final var entityToUpdate = errandsRepository.findById("ERRAND_ID-3");
+		entityToUpdate.get().setCreated(OffsetDateTime.now());
+		errandsRepository.save(entityToUpdate.get());
+
+		final var errandEntities = errandsRepository.findAll(specification, pageable);
+
+		assertThat(errandEntities).isNotNull();
+		assertThat(errandEntities.getTotalElements()).isEqualTo(1);
+
+		assertThat(errandEntities)
+			.extracting(ErrandEntity::getId, ErrandEntity::getAssignedGroupId, ErrandEntity::getAssignedUserId).containsExactlyInAnyOrder(
+				tuple("ERRAND_ID-3", "ASSIGNED_GROUP_ID-3", "ASSIGNED_USER_ID-3"));
+	}
+
+	@Test
+	void shouldThrowExceptionWhenDateTimeIsOfWrongFormat() {
+		//Setup date filter;
+		final var invalidDateFormat = "31-01-2023T16:00:20.954+01:00";
+		final var filter = String.format("(created > '" + invalidDateFormat + "')");
```

Comment:
This is the way String.format should be used:

`String.format("(created > '%s')", invalidDateFormat);`